src/app/[locale]/globals.css

```diff
@@ -7,7 +7,89 @@
 		text-wrap: balance;
 	}
 
+	/* 
+	===============
+	Extra Heading 
+	===============
+	*/
 	.vm-text-eh1 {
 		@apply text-6xl font-semibold leading-[70px] tracking-tight;
 	}
```

Comment:
Using arbitrary values in Tailwind is a no-no. They are a huge performance killer. So, please try to make use of arbitrary values as less as possible, ideally never. Don't forget, these classes you write here, will be applied to all texts in this project. You don't need everything to be pixel-perfect if you have close enough values.

In the names of Typographies, our design has the ratio of line heights. The following is the name for extra heading 1 and you can easily see that it has 120% line height.

`Extra Heading/Eh1 - 58 Semi Bold -  Lh 120%`

From the following link, you can see that Tailwind has some ratio-based line height setters like `leading-none`, `leading-tight` etc. Try to use closer ratios from built-in Tailwind classes. Please apply this change to all the typography classes.

Reply:
Here is the link btw

https://tailwindcss.com/docs/line-height

Reply:
Noted I will make the necessary corrections 
I will take out the arbitrary values and use the ratio based line height instead

Reply:
I have made the necessary changes